code/items/equipments/Hands.cs

```diff
@@ -0,0 +1,191 @@
+using System.Threading.Tasks;
+
+using Sandbox;
+
+using TTTReborn.Globals;
+using TTTReborn.Player;
+
+namespace TTTReborn.Items
+{
+    public interface IGrabbable
+    {
+        bool IsHolding { get; }
+        void Drop();
+        void Update(TTTPlayer player);
+        void SecondaryAction();
+    }
+
+    [Library("equipment_hands")]
+    [Equipment(SlotType = SlotType.UtilityEquipment)]
+    [Hammer.Skip]
+    partial class Hands : TTTEquipment
+    {
+        public override string ViewModelPath => "";
+        public override bool CanDrop() => false;
+        public static readonly float MAX_INTERACT_DISTANCE = 75;
+        public static readonly string MIDDLE_HANDS_ATTACHMENT = "middle_of_both_hands";
+
+        private const float MAX_PICKUP_MASS = 205;
+        private Vector3 MAX_PICKUP_SIZE = new(50, 50, 50);
+        private const float PUSHING_FORCE = 600f;
+
+        private IGrabbable GrabbedEntity;
+        private bool IsHoldingEntity => GrabbedEntity != null && (GrabbedEntity?.IsHolding ?? false);
+        private bool IsPushingPlayer = false;
+
+        public override void Spawn()
+        {
+            base.Spawn();
+
+            RenderColor = Color.Transparent;
+        }
+
+        public override void Simulate(Client client)
+        {
+            base.Simulate(client);
+
+            if (!IsServer)
+            {
+                return;
+            }
+
+            if (Owner is not TTTPlayer player)
+            {
+                return;
+            }
+
+            using (Prediction.Off())
+            {
+                if (Input.Pressed(InputButton.Attack1))
+                {
+                    TryGrabEntity(player);
+                }
+                else if (Input.Pressed(InputButton.Attack2))
+                {
+                    if (IsHoldingEntity)
+                    {
+                        GrabbedEntity?.Drop();
+                    }
+                    else
+                    {
+                        PushPlayer(player);
+                    }
+                }
+                else if (Input.Pressed(InputButton.Reload) && IsHoldingEntity)
+                {
+                    GrabbedEntity?.SecondaryAction();
+                }
```

Comment:
I think it would be more intuitive using the grap action to throw if you're holding an entity, so we replace `InputButton.Reload` with `InputButton.Attack1` moving it with a check into your first scope